### PR TITLE
[ES] Update `People/User_group` and `People/osu!_team`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/osu!_team/es.md [hunk 1]
```diff
@@ -21,22 +20,22 @@ Las personas que se enumeran a continuación son el núcleo del **osu! team** y
...
-| ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Organizador y administrador de la [World Cup](/wiki/Tournaments#official-world-cups) |
-| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Divulgación de [artistas destacados](/wiki/People/Featured_Artists), secretario de la [World Cup](/wiki/Tournaments#official-world-cups) |
+| ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Organizador y administrador de la [World Cup](/wiki/Tournaments#world-cups-oficiales) |
+| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Divulgación de [artistas destacados](/wiki/People/Featured_Artists), secretario de la [World Cup](/wiki/Tournaments#world-cups-oficiales) |
 | ::{ flag=NO }:: [MillhioreF](https://osu.ppy.sh/users/941094) | Desarrollador de osu!, soporte técnico |
-| ::{ flag=JP }:: [nanaya](https://osu.ppy.sh/users/2387883) | Desarrollador de osu!web |
-| ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Encargado general, asesor de moderación |
+| ::{ flag=JP }:: [nanaya](https://osu.ppy.sh/users/2387883) | Desarrollador de la osu!web |
```
Comment (agatemosu) [inline wiki/People/osu!_team/es.md:35]:
```suggestion
| ::{ flag=JP }:: [nanaya](https://osu.ppy.sh/users/2387883) | Desarrollador de osu!web |
```

File: wiki/People/osu!_team/es.md [hunk 2]
```diff
@@ -21,22 +20,22 @@ Las personas que se enumeran a continuación son el núcleo del **osu! team** y
...
+| ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Encargado general, asesor de la moderación |
```
Comment (agatemosu) [inline wiki/People/osu!_team/es.md:36]:
```suggestion
| ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Encargado general, asesor de moderación |
```

File: wiki/People/osu!_team/es.md [hunk 3]
```diff
@@ -21,22 +20,22 @@ Las personas que se enumeran a continuación son el núcleo del **osu! team** y
...
 | ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | Asesor de la escena de modding y mapping |
-| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | Desarrollador de osu!web, [Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
+| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | Desarrollador de la osu!web, [artista destacado](https://osu.ppy.sh/beatmaps/artists/1) |
```
Comment (agatemosu) [inline wiki/People/osu!_team/es.md:38]:
```suggestion
| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | Desarrollador de osu!web, [artista destacado](https://osu.ppy.sh/beatmaps/artists/1) |
```

File: wiki/People/osu!_team/es.md [hunk 4]
```diff
@@ -52,7 +51,7 @@ Las personas que se enumeran a continuación son el núcleo del **osu! team** y
 | ::{ flag=SE }:: [Walavouchey](https://osu.ppy.sh/users/5773079) | [Mantenedor de la wiki](/wiki/People/osu!_wiki_maintainers), editor de noticias |
 | ::{ flag=AU }:: [Zallius](https://osu.ppy.sh/users/55) | Especie en peligro |
 
-El [equipo de soporte de cuentas](/wiki/People/Account_support_team) también está ahí para ayudar con lo que está fuera del alcance.
+El [equipo de soporte de cuentas](/wiki/People/Account_support_team) también está ahí para ayudar con lo que está fuera de alcance.
```
Comment (agatemosu) [inline wiki/People/osu!_team/es.md:54]:
```suggestion
El [equipo de soporte de cuentas](/wiki/People/Account_support_team) también está ahí para ayudar con lo que está fuera del alcance.
```

File: wiki/People/osu!_team/es.md [hunk 5]
```diff
@@ -62,34 +61,34 @@ Además del núcleo del osu! team, hay [muchos otros equipos de voluntarios](/wi
...
+| ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) | Organizador y administrador de la [World Cup](/wiki/Tournaments#world-cups-oficiales), exlíder del [QAT](/wiki/People/Quality_Assurance_Team/QAT_Leaders), administrador de la comunidad |
 | ::{ flag=US }:: [LuigiHann](https://osu.ppy.sh/users/1079) | Diseñador de skins épicas, contribuidor del conjunto de iconos y skins predeterminadas |
 | ::{ flag=DE }:: [Mao](https://osu.ppy.sh/users/2204515) | Exlíder del [NAT](/wiki/People/Nomination_Assessment_Team) |
 | ::{ flag=CA }:: [mm201](https://osu.ppy.sh/users/30655) | Desarrollador de osu!, creador de los sliders mm |
 | ::{ flag=US }:: [Noffy](https://osu.ppy.sh/users/1541323) | Asesor de la escena de modding y mapping |
 | ::{ flag=US }:: [nuudles](https://osu.ppy.sh/users/21312) | Desarrollador de osu! iPhone |
 | ::{ flag=CA }:: [Nyquill](https://osu.ppy.sh/users/682935) | Desarrollador de osu!, organizador del [Mappers' Guild](/wiki/Community/Mappers_Guild) |
-| ::{ flag=DE }:: [p3n](https://osu.ppy.sh/users/123703) | Asesor de moderación |
+| ::{ flag=DE }:: [p3n](https://osu.ppy.sh/users/123703) | Asesor de la moderación |
```
Comment (agatemosu) [inline wiki/People/osu!_team/es.md:82]:
```suggestion
| ::{ flag=DE }:: [p3n](https://osu.ppy.sh/users/123703) | Asesor de moderación |
```

**Review by agatemosu — COMMENTED** [review]
no

File: wiki/People/User_group/es.md [hunk 6]
```diff
@@ -12,16 +12,16 @@ no_native_review: true
...
+Al hacer clic en las insignias, se accederá a las listas del grupo respectivo, mientras que los nombres se vinculan a los artículos de la wiki.
 
 | ID | Insignia | Nombre | Descripción |
 | :-: | :-: | :-- | :-- |
 | 4 | [![GMT](/wiki/shared/group/GMT.png)](https://osu.ppy.sh/groups/4) | [Global Moderation Team](/wiki/People/Global_Moderation_Team) | Vigilar los foros y el chat del juego |
 | 7 | [![NAT](/wiki/shared/group/NAT.png?20231028)](https://osu.ppy.sh/groups/7) | [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) | Administrar a los Beatmap Nominators |
 | 11 | [![DEV](/wiki/shared/group/DEV.png?20231028)](https://osu.ppy.sh/groups/11) | [Developers](/wiki/People/Developers) | Hacer que el juego sea increíble agregando nuevas funciones y corrigiendo los errores |
-| 16 | [![ALM](/wiki/shared/group/ALM.png)](https://osu.ppy.sh/groups/16) | [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que desde entonces se han mudado |
+| 16 | [![ALM](/wiki/shared/group/ALM.png)](https://osu.ppy.sh/groups/16) | [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
```
Comment (agatemosu) [inline wiki/People/User_group/es.md:22]:
```suggestion
| 16 | [![ALM](/wiki/shared/group/ALM.png)](https://osu.ppy.sh/groups/16) | [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que desde entonces se han mudado |
```

File: wiki/People/User_group/es.md [hunk 7]
```diff
@@ -12,16 +12,16 @@ no_native_review: true
...
 | 11 | [![DEV](/wiki/shared/group/DEV.png?20231028)](https://osu.ppy.sh/groups/11) | [Developers](/wiki/People/Developers) | Hacer que el juego sea increíble agregando nuevas funciones y corrigiendo los errores |
-| 16 | [![ALM](/wiki/shared/group/ALM.png)](https://osu.ppy.sh/groups/16) | [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que desde entonces se han mudado |
-| 22 | [![SPT](/wiki/shared/group/SPT.png)](https://osu.ppy.sh/groups/22) | [Support Team](/wiki/People/Support_Team) | Ayuda y asistencia |
-| 28 | [![BN](/wiki/shared/group/BN.png)](https://osu.ppy.sh/groups/28) | [Beatmap Nominators](/wiki/People/Beatmap_Nominators) | Los usuarios van más allá de la llamada para garantizar que los beatmaps se califiquen |
+| 16 | [![ALM](/wiki/shared/group/ALM.png)](https://osu.ppy.sh/groups/16) | [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
+| 22 | [![SPT](/wiki/shared/group/SPT.png)](https://osu.ppy.sh/groups/22) | [Equipo de soporte técnico](/wiki/People/Technical_Support_Team) | Ayuda y asistencia |
```
Comment (agatemosu) [inline wiki/People/User_group/es.md:23]: revierte esto, ya he abierto un pr para esto

**Review by agatemosu — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**